src/components/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './TodoForm.css';
+
+export class TodoForm extends React.PureComponent {
+  state = {
+    todoTitle: '',
+    selectedUser: '',
+    showValueError: false,
+    showUserError: false,
+    userID: this.props.userList.length,
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  }
+
+  handleSubmit = (event) => {
+    const { addTodo } = this.props;
+    const { todoTitle, selectedUser, userID } = this.state;
+
+    event.preventDefault();
+
+    if (!todoTitle.trim() || !selectedUser) {
+      if (!todoTitle.trim()) {
+        this.setState({ showValueError: true });
+        setTimeout(() => (this.setState({ showValueError: false })), 2000);
```

Comment:
I think setTimeout with this.setState is redundant here, you will set these errors to false after adding new todo